classes/event/LanternEvent.php

```diff
@@ -8,6 +8,27 @@ class LanternEvent extends Event {
     public array $item_ids = [];
     public array $mission_ids = [];
```

Comment:
Can we remove these fields now that we have the static fields?

Reply:
This is still accessed in object context as part of the Event class type.